index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="styles/icono.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
+  <title>Chantal Codes</title>
+</head>
+
+<body>
+  <img alt="foggy blue mountains" src="assets/background.jpg" />
+  <header>
+    <nav class="site-nav">
+      <a href="about_me.html">About Me</a>
+      <a href="portfolio.html">Portfolio</a>
+      <a href="hobby_blog.html">Hobby-Blog</a>
+      <a href="index.html">Home</a>
+    </nav>
+  </header>
+
+
+  <section>
+
+    <article>
+      <h3>Chantal Rollison</h3>
+      <div class="short-bio">
+        <p>
+          Thanks for visiting! By now you may know that I am a student at
+          <a href="http://adadevelopersacademy.org/" target="_blank">Ada Developer's Academy</a>, preparing
+          to be a junior developer.
+        </p>
+
+        <p>
+          What you may not know is that most of my formal education is in
+          Biology and Accounting. It's my non-traditional background that
+          gives me a unique world view, and one that I believe will benefit
+          the tech industry.
+        </p>
+
+        <p>
+          Have a look around and drop me a line! If you're shy, you're welcome
+          to stalk me on Google.
+        </p>
+
+        <section>
```

Comment:
Could this be part of the footer?  Because your footer is fixed and the icons here have a container (the section) that is inheriting the relative position , they end up on top of each other:

![image](https://cloud.githubusercontent.com/assets/13111136/24519709/d22dc9ee-153a-11e7-9946-465bbf48c160.png)

You could fix this by assigning the section element a position of absolute and then messing around with it's distance from the bottom.  I think you would also have to adjust the bottom padding on your short-bio div so your content could change (get bigger or smaller) and not cause a collision with your icons.  Or you could add these icons to the footer (but then you would always be able to see them).

I played around with the elements in the Dev Tools thing (whatever it is that you get when you inspect the elements - you mentioned it in your summary) to figure this out.  It didn't just come from my head!

I love the Dev Tools for messing with CSS - one helpful tip is there is a lot of changes you can make in the CSS of the dev tools itself to see how it looks. When we first started I though I had to update my CSS and refresh the page all the time. For example, if you click into a positioning number and hit your up/down keys, it's like moving the element around on the screen with arrows.  You don't have to guess at changes like 20 px to 30 px... the up/down keys will show you 21px, 22px, 23px, and so on (I hope this makes sense!). You can also click on one of the color boxes (in the below shot I clicked on white for your font) and it brings up an interactive color picker.  You can pick the color from the provided gradient, or it also give you like a magnifying glass thing and you can pick the color from elsewhere on the page.

![image](https://cloud.githubusercontent.com/assets/13111136/24520268/b4d5138c-153c-11e7-8fcd-3f03fdbcb0b3.png)

It. is. AMAZING.  If you already know this I am going to be so mad because it definitely feels to me like secret inside information I am disseminating right now.

So after you've got it perfect, you can just copy the CSS and paste it into your file :).